ntap_annotations/ntap_ncats_annotation_scheme.yml

```diff
@@ -41,8 +41,8 @@ entityId1: syn5611797, syn5611796
 filePath1: '10x10 Combination Screens/HFF'  ##this is an alternate way of referencing an entity if localOrRemote is remote, than file path refers to head of synapse project.
      annotations:
```

Comment:
No, let's just leave it as it is because Xindi has already built a script to create a manifest around it.